hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/FairSchedulerAppsBlock.java

```diff
@@ -129,6 +129,12 @@ protected Boolean hasAccess(RMApp app, HttpServletRequest hsr) {
     return true;
   }
 
+  private static String printAppInfo(long appInfoFunc) {
```

Comment:
You are passing a long value which is representing some numerical value (containers, memory MB, vcores) so the name of the parameter should represent it correctly. I suggest 'value' as it's generic enough. 
'appInfoFunc' is not correct as the values you are passing are not functions

Reply:
yes I agree, will do the changes.